cmd/fyne/commands/deprecated.go

```diff
@@ -0,0 +1,45 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime/debug"
+
+	"github.com/urfave/cli/v2"
+)
+
+// Vendor returns the vendor cli command
+//
+// Deprecated: Use "go mod vendor" instead
+func Vendor() *cli.Command {
+	return &cli.Command{
+		Name:  "vendor",
+		Usage: "Deprecated: Use \"go mod vendor\" instead.",
+		Action: func(_ *cli.Context) error {
+			cmd := exec.Command("go", "mod", "vendor")
+			cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
+			return cmd.Run()
+		},
+	}
+}
+
+// Version returns the cli command for the program version
+//
+// Deprecated: Use "go mod vendor" instead
```

Comment:
This deprecation comment is inconsistent with the usage instructions for this command.

```suggestion
// Deprecated: Use "fyne --version" or "fyne -v" instead.
```